src/pages/index.js

```diff
@@ -0,0 +1,380 @@
+import {
+  enableValidation,
+  validationConfig,
+  resetValidation,
+} from "../scripts/validation.js";
+import "../pages/index.css";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c1da3710-50e8-4d77-9af1-7ab507c7d59c",
+    "Content-Type": "application/json",
+  },
+});
+
+//destrucure the second item in the callback of the .then()
+api
+  .getAppInfo()
+  .then(([userInfo, cards]) => {
+    console.log(cards);
+
+    // render card
+    cards.forEach((card) => renderCard(card, "append"));
+    // initialCards.forEach((item) => renderCard(item, "append"));
+
+    const avatarElement = document.querySelector(".profile__avatar");
+    const nameElement = document.querySelector(".profile__name");
+    const aboutElement = document.querySelector(".profile__about");
+
+    if (userInfo) {
+      avatarElement.src = userInfo.avatar; // Set avatar image
+      nameElement.textContent = userInfo.name; // Set user's name
+      //   aboutElement.textContent = userInfo.about; // Set user's bio/description
```

Comment:
* When I reload the page, the **old job** description appears again


Please, use it to set the job